ocs_ci/helpers/dr_helpers.py

```diff
@@ -503,19 +546,28 @@ def wait_for_replication_resources_deletion(namespace, timeout, check_state=True
         TimeoutExpiredError: In case replication resources not deleted
 
     """
+    # TODO: Improve the parameter for condition
+    if "cephfs" in namespace:
+        resource_kind = constants.REPLICATION_SOURCE
+        count_function = get_replicationsources_count
+    else:
+        resource_kind = constants.VOLUME_REPLICATION
```

Comment:
Same

Reply:
resource_kind = constants.VOLUME_REPLICATION is for RBD.

Reply:
yeah